scripts/core/ui/components/ListPage/generic-list-page.tsx

```diff
@@ -613,12 +601,33 @@ export class GenericListPageComponent<T extends IBaseRestApiResponse>
     }
 }
 
-export const getGenericListPageComponent = <T extends IBaseRestApiResponse>(resource: string) =>
-    connectServices<IPropsGenericForm<T>>(
-        connectCrudManager<IPropsGenericForm<T>, IPropsConnected<T>, T>(
-            GenericListPageComponent,
-            'items',
-            resource,
-        )
-        , ['modal', '$rootScope', 'notify'],
-    );
+export const getGenericListPageComponent =
+    <T extends IBaseRestApiResponse>(resource: string, formConfig?: IFormGroup) =>
+        connectServices<IPropsGenericForm<T>>(
+            connectCrudManager<IPropsGenericForm<T>, IPropsConnected<T>, T>(
+                GenericListPageComponent,
```

Comment:
It would be better if the component returned from `getGenericListPageComponent` wouldn't have a `formConfig` prop and would use the one passed to function as a parameter, but it's complex to do, so don't bother with it. I just wanted to mention a possible improvement.